src/main/java/com/doeiqts/pizzavoter/servlet/PizzaVoterServlet.java

```diff
@@ -138,17 +137,22 @@ private Pizza serializePizza(HttpServletRequest request, String pizzaNumber) {
                     Sauce.valueOf(request.getParameter("sauce" + pizzaNumber)));
 
             String[] toppings = request.getParameterValues("toppings" + pizzaNumber);
-            String[] positions = request.getParameterValues("position" + pizzaNumber);
             for (int i = 0; i < toppings.length; i++) {
-                if (toppings[i] != "") {
-                    Position newPosition = Position.valueOf(positions[i]);
-                    Position old = pizza.addTopping(Topping.valueOf(toppings[i]),newPosition);
-                    if(Position.ALL.equals(old) || (Position.RIGHT.equals(old) && newPosition.equals(Position.LEFT)) ||
-                        (Position.LEFT.equals(old) && newPosition.equals(Position.RIGHT))) {
-                        pizza.addTopping(Topping.valueOf(toppings[i]), Position.ALL);
+                if (toppings[i] != "" && pizza.getToppingCount() < Pizza.getToppingLimit()) {
```

Comment:
Do we need to check the limits here if the domain object already handles that logic?